test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: false
+
+require_relative '../helper'
+
+require 'csv'
+
+class TestFilter < Test::Unit::TestCase
+
+  # Some rows data (useful as default).
+  Rows = [
+    %w[aaa bbb ccc],
+    %w[ddd eee fff],
+  ]
```

Comment:
Could you use `setup` and instance variable instead?

```suggestion
  def setup
    @rows = [
      %w[aaa bbb ccc],
      %w[ddd eee fff],
    ]
  end
```

Reply:
Modified.